File: Data Structures & Algorithms/number-of-senior-citizens/submission-0.java
class Solution {
    public int countSeniors(String[] details) {
        int count = 0;
        for(int i = 0; i < details.length; i++){
            String string = new String(details[i].substring(11,13));
            int age = Integer.parseInt(string);
            if (age > 60) {
                count += 1;
            }
        }
        return count;
    }
}

File: Data Structures & Algorithms/kth-distinct-string-in-an-array/submission-0.py
class Solution:
    def kthDistinct(self, arr: List[str], k: int) -> str:
        count = {}
        for string in arr:
            count[string] = 1 + count.get(string, 0)
        
        for string in arr:
            if count[string] == 1:
                k -= 1
            if k == 0:
                return string
        return ""
